src/app.ts

```diff
@@ -18,6 +19,7 @@ import { NumberMap } from './types/types'
 import { log, logToServer } from './util/logger'
 import { initInfoServer } from './util/network'
 
+// LogBox.ignoreAllLogs()
```

Comment:
Leftover comment?

Reply:
Actually. I should have added a comment. I intentionally added that to easily disable the logbox.